src/overlays/actors/ovl_En_Dnk/z_en_dnk.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_en_dnk.c
+ * Overlay: ovl_En_Dnk
+ * Description: Hallucinatory Mad Scrubs
```

Comment:
Maybe mention that these are in the Deku curse and healing cutscenes... I had to think for a minute to crack that one.